esp32_ulp/opcodes.py

```diff
@@ -354,7 +354,9 @@ def get_cond(arg):
 def _soc_reg_to_ulp_periph_sel(reg):
     # Map SoC peripheral register to periph_sel field of RD_REG and WR_REG instructions.
     ret = 3
-    if reg < DR_REG_RTCCNTL_BASE:
+    if reg < 0x3ff:
+        ret = 0
```

Comment:
can you check whether there is a constant we can use instead of 0 and 3?

Reply:
Will do. The ret = 3 is actually has no use (ret is always assigned something else), so I have dropped it. Also found a constant name for 0x3ff in the binutils-esp32ulp sources (DR_REG_MAX_DIRECT), so switched to using that too.